spp_service_points/views/main_view.xml

```diff
@@ -0,0 +1,19 @@
+<odoo>
+    <!-- Top menu item -->
+
+    <menuitem
+        id="service_points_main_menu_root"
+        name="Service Point"
+        web_icon="spp_idqueue,static/src/img/icons/contacts.png"
```

Comment:
@gonzalesedwin1123 I don't know why you use this web icon, but since we are not using any theme it is fine 🙄